starter-code/routes/welcome.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+
+
+
+//GET home page
+router.get("/", function(req, res, next){
```

Comment:
this route should have an authorization block to disallow anonymous users

even better, a `router.use(...)` block at the top of the file, defining a middleware that runs before the route

```
router.use(function (req, res, next) {
  if (req.session.currentUser) {
    next(); // we have a user in the session, continue executing routes
  } else {
    res.redirect('/'); // no user in session, redirect to login
  }
});
```